src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -471,6 +474,60 @@ public IActionResult UpdateLogLevel([FromBody] LogRulesRequest request)
             }
         }
 
+        /// <summary>
+        /// Get the enabled log rules.
+        /// </summary>
+        /// <returns>A list of log rules.</returns>
+        [HttpGet]
+        [Route("logrules")]
+        public IActionResult GetLogRules()
+        {
+            // Checks the request is valid.
+            if (!this.ModelState.IsValid)
+            {
+                return ModelStateErrors.BuildErrorResponse(this.ModelState);
+            }
+
+            try
+            {
+                var rules = new List<LogRuleModel>();
+
+                foreach (LoggingRule rule in LogManager.Configuration.LoggingRules)
+                {
+                    string filename = string.Empty;
+
+                    // Retrieve the full path of the current rule's log file.
+                    if (rule.Targets.Any() && rule.Targets.First().GetType().Name == "AsyncTargetWrapper")
+                    {
+                        WrapperTargetBase wrapper = (WrapperTargetBase) rule.Targets.First();
+
+                        if (wrapper.WrappedTarget != null && wrapper.WrappedTarget.GetType().Name == "FileTarget")
+                        {
+                            filename = ((FileTarget) wrapper.WrappedTarget).FileName.ToString();
+                        }
+                    }
+                    else if (rule.Targets.Any() && rule.Targets.First().GetType().Name == "FileTarget")
+                    {
+                        filename = ((FileTarget)rule.Targets.First()).FileName.ToString();
+                    }
+
+                    rules.Add(new LogRuleModel
```

Comment:
If `fileName` is null do we want to add a `LogRuleModel` instance to the list?

Reply:
yeah because we don't necessarily only have file targets.
What I can do though is not include the filename in the returned json.